src/components/CommentItem.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from "react";
+import styled from "styled-components";
+
+import { Comment } from "../types";
+import { formatDate } from "../utils/date_utils";
```

Comment:
\[Out of PR scope / optional]: It’d be nice to stick with a consistent convention for file names, which includes letter casing. A common style for JS projects is `camelCase`/`PascalCase` or `kebab-case`. I’ve already put some stuff into `src/shared/helpersForComments.ts` on `main`, so we can either use that file or create a sibling file (`helpersForDates`).

Switching to `util` / `utils` is fine with me too. We just need to be aware that people use two words interchangeably, which make things a bit inconsistent.

Reply:
Got it, will check and update

Reply:
Camel case usually less readable and traditionally more affected to be damaged by the case insensitive system.
Compare: CommentItem and comment-item
How about use kebab case and never use camel case for all files?

Reply:
I’m generally open to `kebab-case` as a project-wide rule. We do need to use `kebab-case` in `src/pages` anyway, because it maps to page URLs.

The main thing that stops me from doing this in own projects is React’s decision to use `PascalCase` for components, even in new docs: https://beta.reactjs.org/learn

<img width="481" alt="Screenshot 2021-12-13 at 00 19 38" src="https://user-images.githubusercontent.com/608862/145735668-c6e5cb2f-dccf-4610-8daa-3cac3bda4407.png">

They would not do this out of nothing and I believe that there exists some research backing the decision. It might be to do with human cognition of text, I don’t know.

If you are more comfortable with `kebab-case`, let’s switch to it everywhere in a follow up PR. I like simple rules and _`kebab-case` everywhere_ is dead simple. We can always change to something else if we face problems 🙂